### PR TITLE
net-p2p/classified-ads: Upstream v0.14 version bump.

### PR DESCRIPTION
**Pull request by operatornormal**
Closes: https://bugs.gentoo.org/836741

Signed-off-by: Antti Järvinen <antti.jarvinen@katiska.org>

### CODE REVIEW

File: net-p2p/classified-ads/classified-ads-0.14-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		popd > /dev/null || die
+	fi
+	if use test; then
+		emake -C testca
+	fi
+}
+
+src_test() {
+	# testca will return 0 if all unit tests pass
+	virtx ./testca/testca
```
Comment (thesamesam) [inline net-p2p/classified-ads/classified-ads-0.14-r1.ebuild:76]: I think virtx is run nonfatal so may need || die here

File: net-p2p/classified-ads/classified-ads-0.14-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	dev-qt/qtwidgets:5
+	media-libs/opus
+	net-libs/libnatpmp
+	net-libs/miniupnpc
+	sys-apps/file
+	sys-libs/zlib
+	virtual/libintl"
+
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[dot] )
```
Comment (thesamesam) [inline net-p2p/classified-ads/classified-ads-0.14-r1.ebuild:40]: doc is BDEPEND

File: net-p2p/classified-ads/classified-ads-0.14-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit qmake-utils virtualx xdg
```
Comment (juippis) [inline net-p2p/classified-ads/classified-ads-0.14-r1.ebuild:5]:
```suggestion
EAPI=8

inherit qmake-utils virtualx xdg
```

We like to keep an empty line for consistency here.

File: net-p2p/classified-ads/classified-ads-0.14-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5[widgets]
+	dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsql:5[sqlite]
+	dev-qt/qtwidgets:5
+	media-libs/opus
+	net-libs/libnatpmp
+	net-libs/miniupnpc
```
Comment (juippis) [inline net-p2p/classified-ads/classified-ads-0.14-r1.ebuild:34]:
```suggestion
	net-libs/miniupnpc:=
```

File: net-p2p/classified-ads/classified-ads-0.14-r1.ebuild [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+	fi
+}
+
+src_compile() {
+	emake
+	if use doc; then
+		pushd doc > /dev/null || die
+		doxygen || die
+		popd > /dev/null || die
+	fi
```
Comment (juippis) [inline net-p2p/classified-ads/classified-ads-0.14-r1.ebuild:68]: This seems so simple maybe using *docs.eclass* could benefit here, but no need to do it now as this works.

File: net-p2p/classified-ads/classified-ads-0.14-r1.ebuild [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+src_test() {
+	# testca will return 0 if all unit tests pass
+	virtx ./testca/testca || die
+}
+
+src_install() {
+	docompress -x /usr/share/doc/
```
Comment (juippis) [inline net-p2p/classified-ads/classified-ads-0.14-r1.ebuild:80]: Maybe you can solve this upstream? Generally package managers like to handle this.

**Review by juippis — APPROVED** [review]
This is good otherwise, but you shouldn't have a revision -r1 for a new version bump. Revisions are used for ebuild enhancements and this PR's history won't be merged to our git history, so you can just force-push the clean version bump here.

**Review by juippis — APPROVED** [review]
Thanks!

**Comment by juippis** [comment]
e3fe8d6581e5257e2ded8028ab355a12822f2c7a

**Pull request closed without merging**